test/translator/translation_test.cc

```diff
@@ -26,15 +26,15 @@ TEST(Translation, ShouldHandleUnknownOpcode) {
   translator::Translation translation = {};
   uint8_t opc = 233;
   translator::Span<const uint8_t> scsi_cmd = translator::Span(&opc, 1);
-  translator::BeginResponse resp = translation.Begin(scsi_cmd, 0);
+  translator::BeginResponse resp = translation.Begin(scsi_cmd, scsi_cmd, 0);
```

Comment:
Is it sensible to pass in the same buffer for both arguments here? If this case isn't expected to read from the second buffer, then a default-constructed empty span might make more sense.